tests/cases/fourslash/quickInfoCanBeTruncated.ts

```diff
@@ -513,7 +513,7 @@
 //// type DeeplyMapped/*6*/ = {[K in keyof Foo]: {[K2 in keyof Foo]: [K, K2, Foo[K], Foo[K2]]}}
 
 goTo.marker("1");
-verify.quickInfoIs(`type A = "_0" | "_1" | "_2" | "_3" | "_4" | "_5" | "_6" | "_7" | "_8" | "_9" | "_10" | "_11" | "_12" | "_13" | "_14" | "_15" | "_16" | "_17" | "_18" | "_19" | "_20" | "_21" | "_22" | "_23" | "_24" | ... 474 more ... | "_499"`);
+verify.quickInfoIs(`type A = keyof Foo`);

```

Comment:
Nice, though I wonder if this affects how people will learn about what `keyof` desugars into.

Reply:
Yeah, you kind of want both. Except that you don't once you understand what's going on.